plugins/spark/README.md

```diff
@@ -1,2 +1,19 @@
-# Spark Plugin
+# Spark Jobs as Flyte Tasks/Workflows
 
+This section provides examples of how to author spark tasks and workflows using FlyteKit as well as additional setup required to run Spark Jobs via Flyte.
+
+## Setup
+1. For Spark, the image must contain spark dependencies as well as the correct entrypoint for the Spark driver/executors. This can be achieved by using
+the `flytekit_install_spark.sh` script provided via flytekit as referenced in the Dockerfile included here.
```

Comment:
can we make this a link to flytekit's script?

Reply:
done